src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java

```diff
@@ -89,22 +92,28 @@ public static void setHttpTimeouts(String seconds) throws Exception {
    * @throws SCHttpUnknownException if an unknown error occurs during the request
    * @return http response string
    */
-  private static String httpRequest(String url, String header, String body, String method)
+  private static String httpRequest(
+      String url, String header, String body, String method, String outputFile)
       throws EWException, IOException, SCHttpEwonException, SCHttpAuthException,
           SCHttpConnectionException, SCHttpUnknownException {
     // Reset counter if it reaches max
     if (tempResponseFileNameCounter >= Integer.MAX_VALUE) {
       tempResponseFileNameCounter = 0;
     }
 
-    // Create file for storing response
-    final File responseFile =
-        new File(
-            TEMP_RESPONSE_FILE_FOLDER
-                + TEMP_RESPONSE_FILE_NAME_PREFIX
-                + tempResponseFileNameCounter++
-                + "."
-                + method);
+    // Create file for storing response (and delete if it already exists)
```

Comment:
This comment is a bit confusing and I am not sure if the () makes it more clear.

Reply:
Updated with your suggestion